test/unit/classes/accessors.spec.ts

```diff
@@ -125,37 +125,6 @@ test("set accessor in base class", () => {
     `.expectToMatchJsResult();
 });
 
-test("set accessor override", () => {
```

Comment:
Could you check skipped tests tests, I think they shouldn't be relevant now too

Reply:
Yep they were too, removed them as well